plugins/auth-backend/api-report.md

```diff
@@ -650,6 +650,8 @@ export interface RouterOptions {
   // (undocumented)
   providerFactories?: ProviderFactories;
   // (undocumented)
+  tokenKeyAlgorithm?: string;
```

Comment:
Maybe we can change this name to `tokenFactoryAlgorithm` instead as it's being passed to the `tokenFactory`?